build/Jenkinsfile

```diff
@@ -128,23 +160,25 @@ podTemplate(label: 'jenkins-slave',
     }
     stage('Style Checks') {
         container('golang') {
-            sh 'cd /go/src/github.com/mattermost/mattermost-server && make check-style BUILD_NUMBER=$BUILD_NUMBER'
+            sh "cd /go/src/github.com/mattermost/mattermost-server && make check-style BUILD_NUMBER=${env.BUILD_NUMBER}"
         }
     }
     stage('Build') {
         container('golang') {
-            sh 'cd /go/src/github.com/mattermost/mattermost-server && make build BUILD_NUMBER=$BUILD_NUMBER'
+            sh "cd /go/src/github.com/mattermost/mattermost-server && make build BUILD_NUMBER=${env.BUILD_NUMBER}"
         }
         container('mattermost-node') {
-            sh 'cd mattermost-webapp && curl -f -o ./dist.tar.gz https://releases.mattermost.com/mattermost-webapp/commit/`git rev-parse HEAD`/mattermost-webapp.tar.gz && mkdir ./dist && tar -xvf ./dist.tar.gz -C ./dist --strip-components=1 || make build'
+            sh "cd mattermost-webapp && curl -f -o ./dist.tar.gz https://releases.mattermost.com/mattermost-webapp/commit/`git rev-parse HEAD`/mattermost-webapp.tar.gz && mkdir ./dist && tar -xvf ./dist.tar.gz -C ./dist --strip-components=1 || make build BUILD_NUMBER=${env.BUILD_NUMBER}"
```

Comment:
Is there a reason you're putting this here? If the webapp repo's `make build` actually uses this build number, this caching just won't work at all and I need to revert it entirely.

Reply:
my bad
removed the build num